MBBSEmu.Tests/ExportedModules/Majorbbs/cncchr_Tests.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using MBBSEmu.Memory;
+using Xunit;
+
+namespace MBBSEmu.Tests.ExportedModules.Majorbbs
+{
+    public class cncchr_Tests : MajorbbsTestBase
+    {
+        private const int CNCCHR_ORDINAL = 122;
+
+        [Theory]
+
+        //Simple commands with nxtcmd == input
+        [InlineData("123", 1, '2', 2)]
+        [InlineData("1 2", 0, '1', 2)]
+        [InlineData("2 1", 2, '1', 3)]
+        [InlineData("123 456", 4, '4', 5)]
```

Comment:
additional cases: "123          456" , .... basically the same thing, just with excessive spacing between the two chars

InlineData("123", 3, '\0', 3)

Reply:
Oh good call -- I'll add the test and the logic to handle multiple spaces